iris_grib/tests/unit/save_rules/test_identification.py

```diff
@@ -14,25 +14,26 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with iris-grib.  If not, see <http://www.gnu.org/licenses/>.
-"""Unit tests for `iris.fileformats.grib.grib_save_rules.identification`."""
+"""Unit tests for `iris_grib.grib_save_rules.identification`."""
 
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-# Import iris.tests first so that some things can be initialised before
+# Import iris_grib.tests first so that some things can be initialised before
 # importing anything else.
-import iris.tests as tests
+import iris_grib.tests as tests
 
 import gribapi
 
-import iris.fileformats.grib
-from iris.fileformats.grib._save_rules import identification
-from iris.tests import mock
+import iris
```

Comment:
It's using `iris.coords.AuxCoord` to supplement an iris stock cube. I wouldn't class that as an integration test.

But I agree that we should be careful.